### PR TITLE
update Participate in Governance overview for governance section revamp

### PR DESCRIPTION
**Pull request by thenic95**
## Checklist

- [x] I have read the [How to Contribute](https://developers.cardano.org/docs/portal-contribute/).
- [x] I have run `yarn build` after adding my changes **without getting any errors**. 

## Updating documentation or Bugfix

Rewrite the [Overview.md](https://developers.cardano.org/docs/governance/) file of the Main Category to reflect the current status quo of Governance accurately. 
As indicated in #1223 

After lot's of trial and error I made it work @rphair 😄

### CODE REVIEW

**Review by rphair — APPROVED** [review]
Looks good to me... maybe @Ryun1 would like to confirm before we merge?

File: docs/governance/overview.md [hunk 1]
```diff
@@ -1,26 +1,26 @@
 ---
 id: overview
 slug: /governance/
-title: Participate in Governance 
+title: Participate in Governance
 sidebar_label: Overview
-description: Governance is an essential topic that includes Cardano Improvement Proposals (CIP), project funding, voting, and network parameters.
+description: Governance is an essential topic that includes the on-chain governance framework, Cardano Improvement Proposals (CIP), Cardano Problem Statements (CPS), Project Catalyst, and Network Parameters.
```
Comment (Ryun1) [inline docs/governance/overview.md:6]:
```suggestion
description: Governance is an essential topic that includes the on-chain protocol governance, Cardano Improvement Proposals (CIP), Cardano Problem Statements (CPS), Project Catalyst, and Network Parameters.
```
I think "protocol governance" is a more specific term

File: docs/governance/overview.md [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
 image: /img/og/og-developer-portal.png
 ---
 
 ![Cardano Governance](../../static/img/card-governance-title.svg)
 
-Cardano was designed with the intent of long-term sustainability in mind. The goal was to build a system that could govern and fund itself without relying on donations. Cardano includes a built-in treasury system that receives revenue via transaction fees and treasury taxes on a regular basis. This is explained in detail in [treasuries with Bingsheng Zhang](https://www.youtube.com/watch?v=Hyh3h_yX-S0). 
+Cardano has been meticulously designed to ensure sustainability and longevity. A cornerstone of this vision is its self-sufficient governance and funding model, which eliminates the need for external funding. At the heart of Cardano's innovative approach is an on-chain treasury system, consistently funded through transaction fees and treasury taxes. These mechanisms are pivotal for the platform's ongoing development. They are detailed in discussions with [Bingsheng Zhang](https://www.youtube.com/watch?v=Hyh3h_yX-S0), providing valuable insights into the treasury's operations.
 
-## Cardano Improvement Proposal
+## Cardano Governance
```
Comment (Ryun1) [inline docs/governance/overview.md:14]:
```suggestion
## Cardano Protocol Governance
```

File: docs/governance/overview.md [hunk 3]
```diff
@@ -1,26 +1,26 @@
...
 image: /img/og/og-developer-portal.png
 ---
 
 ![Cardano Governance](../../static/img/card-governance-title.svg)
 
-Cardano was designed with the intent of long-term sustainability in mind. The goal was to build a system that could govern and fund itself without relying on donations. Cardano includes a built-in treasury system that receives revenue via transaction fees and treasury taxes on a regular basis. This is explained in detail in [treasuries with Bingsheng Zhang](https://www.youtube.com/watch?v=Hyh3h_yX-S0). 
+Cardano has been meticulously designed to ensure sustainability and longevity. A cornerstone of this vision is its self-sufficient governance and funding model, which eliminates the need for external funding. At the heart of Cardano's innovative approach is an on-chain treasury system, consistently funded through transaction fees and treasury taxes. These mechanisms are pivotal for the platform's ongoing development. They are detailed in discussions with [Bingsheng Zhang](https://www.youtube.com/watch?v=Hyh3h_yX-S0), providing valuable insights into the treasury's operations.
```
Comment (Ryun1) [inline docs/governance/overview.md:12]: > and treasury taxes

what does this mean?

File: docs/governance/overview.md (hunk 1 above)
Comment (thenic95) [reply docs/governance/overview.md:6]: I like it. Thanks for the suggestion.

File: docs/governance/overview.md (hunk 3 above)
Comment (thenic95) [reply docs/governance/overview.md:12]: I'm here referring to the `τ` parameter, the treasury growth rate. 
Thinking about it I should probably rephrase this sentence. Here is my suggestions:

> At the heart of Cardano's innovative approach is an on-chain treasury system, consistently funded each epoch by a fixed percentage of the virtual pot, which encompasses all transaction fees collected within that epoch and the ada allocated from the reserve, with the reserve's ada allocation determined by the monetary expansion rate.

Comment (Ryun1) [reply docs/governance/overview.md:12]: This sounds good

The term `treasury taxes` was confusing for me.

**Comment by rphair** [comment]
thanks @thenic95 @Ryun1 for the coordinated editing 🤩

**Pull request merged**